newrelic/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>beadledom-parent</artifactId>
+    <groupId>com.cerner.beadledom</groupId>
+    <version>2.8-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>beadledom-newrelic</artifactId>
```

Comment:
Could you ever foresee using new relic features from processing projects or other projects that aren't HTTP services?

Reply:
Probably?? I really don't know. Do you think we should add something around "services/jaxrs" to the artifactId.

Reply:
Maybe it could be `beadledom-newrelic-jaxrs`, then in the future if the `NewRelicApi` class got reused somewhere outside of an HTTP service, those classes could be extracted to a `beadledom-newrelic` module that is used both by this module and others? It's hard to say if it's worth it.

Reply:
I agree. Refactored to `beadledom-newrelic-jaxrs` https://github.com/cerner/beadledom/pull/93/commits/fc24fb547e41046d5ab4d65310b2a38e00e771e7